### PR TITLE
Fixes/usage instructions

### LINKED ISSUE #1: Usage section is empty
How can we use this gem?

### PR DESCRIPTION
**Pull request by mauro-oto**
Hey @etagwerker, @schmierkov, 

This PR fixes #1. I added some basic usage instructions, should be enough to get someone new started.

Please check it out, thanks!

### CODE REVIEW

File: lib/bitpagos/client.rb [hunk 1]
```diff
@@ -22,42 +22,67 @@ def get_transaction(transaction_id)
...
     end
 
     def waiting_transactions
-      retrieve_transactions(WAITING)
+      retrieve_transactions({status: WAITING})
     end
 
     def pending_transactions
-      retrieve_transactions(PENDING)
+      retrieve_transactions({status: PENDING})
```
Comment (etagwerker) [inline lib/bitpagos/client.rb]: @mauro-oto The curly braces are not necessary when you write lines like this one. I'd recommend you get rid of them. ;)

File: lib/bitpagos/client.rb [hunk 2]
```diff
@@ -22,42 +22,67 @@ def get_transaction(transaction_id)
...
-      all_transactions["meta"]["total_count"]
+      transactions["meta"]["total_count"]
     end
 
     private
 
+    # Takes a symbol and returns the proper transaction type.
+    #
+    # @param [Symbol] Can be :pending, :waiting, :completed or :partially_paid
+    # @return [String] Returns the corresponding "PE", "WA", "CO" or "PP"
```
Comment (etagwerker) [inline lib/bitpagos/client.rb]: @mauro-oto I guess this method could return `nil` too. If so, the documentation should say: 

```ruby
 @return [String,nil]
```

**Comment by etagwerker** [comment]
@mauro-oto Looks good! Just added those two comments.

**Comment by mauro-oto** [comment]
@etagwerker just updated the PR based on your comments. Thanks!

**Comment by etagwerker** [comment]
@mauro-oto :+1:

**Pull request merged**